_rules/important-letter-spacing-wide-enough-24afc2.md

```diff
@@ -16,40 +16,48 @@ input_aspects:
 acknowledgments:
   authors:
     - Jean-Yves Moyen
+  previous_authors:
     - Jey Nandakumar
   funding:
     - WAI-Tools
 ---
 
 ## Applicability
 
-This rule applies to any [HTML element][] that is [visible][] and for which the `style` attribute [declares][declared] the [letter-spacing][] CSS property.
+This rule applies to any [visible][] [text node][] child of an [HTML element][], when all the following are true for its `letter-spacing` property:
+
+- the [specified][] value is [declared][] in a `style` attribute; and
+- the [computed][] value is [important][].
+
+Where the `letter-spacing` property of a text node means the one of its parent.
```

Comment:
I think you can move this to the background, rather than repeat this in the applicability and expectation. I think this is sufficiently unambiguous on its own.